src/main/java/com/wolox/training/exceptions/ErrorResponse.java

```diff
@@ -0,0 +1,20 @@
+package com.wolox.training.exceptions;
+
+public class ErrorResponse {
+
+    private String reason;
+    private Integer code;
```

Comment:
The same in this case. The fields have to be finals.

Reply:
Thanks Mati! I did the changes!